python/fusion_engine_client/parsers/mixed_log_reader.py

```diff
@@ -76,6 +77,9 @@ def __init__(self, input_file, warn_on_gaps: bool = False, show_progress: bool =
             if len(self.message_types) == 0:
                 self.message_types = None
 
+        # The source IDs requested by the user that are available.
+        self.requested_source_ids = source_ids
```

Comment:
Convert the list to a set here so you can treat it as a set later for taking intersections